plan/ranger/range.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package plan
+package ranger
```

Comment:
It's better to move ranger to util package. Because statistic package still need to use it.